src/index.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import 'materialize-css';
```

Comment:
I wonder what the difference between 
`import 'materialize-css';` and `import 'materialize-css/dist/css/materialize.min.css';`
Looks like one is included in the other to me

Reply:
@j33n `import 'materialize-css';`  installs the JS asset only